Website/Src/Views/Manage/Index.cshtml

```diff
@@ -152,6 +152,17 @@
             </dd>
         </dl>
         <dl class="dl-horizontal">
+            <dt>
+                <label asp-for="PreferDarkTheme" class="control-label"></label>
+            </dt>
+            <dd class="checkbox">
+                <label asp-for="PreferDarkTheme">
+                    <input type="checkbox" asp-for="PreferDarkTheme" />
```

Comment:
Similar to above, could we do a radio instead of checkbox?

Reply:
done